neural_compressor/torch/quantization/config.py

```diff
@@ -503,6 +506,76 @@ def get_default_sq_config() -> SmoothQuantConfig:
     return SmoothQuantConfig()
 
 
+######################## HQQ Config ###############################
+@register_config(framework_name=FRAMEWORK_NAME, algo_name=HQQ, priority=PRIORITY_HQQ)
+class HQQConfig(BaseConfig):
+    # Half-Quadratic Quantization (HQQ), more details:
+    # https://mobiusml.github.io/hqq_blog/
+    # Ported from https://github.com/mobiusml/hqq
+
+    name = HQQ
+    params_list = [
+        "nbits",
+        "group_size",
+        "quant_zero",
+        "quant_scale",
+        "scale_quant_group_size",
```

Comment:
Is it a kind of double_quant?

Reply:
Yes, the official HQQ supports quant both `zero` and `scale`. However, we have not fully explored these configurations yet. I would prefer to maintain the official implementation as is and update it as we go along.